src/BaGet/Migrations/PostgresSql/20181206131511_Initial.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+using Npgsql.EntityFrameworkCore.PostgreSQL.Metadata;
+
+namespace BaGet.Migrations.PostgresSql
+{
+    public partial class Initial : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
+                name: "Packages",
+                columns: table => new
+                {
+                    Key = table.Column<int>(nullable: false)
+                        .Annotation("Npgsql:ValueGenerationStrategy", NpgsqlValueGenerationStrategy.SerialColumn),
+                    Id = table.Column<string>(maxLength: 128, nullable: false),
```

Comment:
The `Package.Id` and `PackageDependency.Id` columns must be case insensitive. See [`SqliteContext.OnModelCreating`](https://github.com/loic-sharma/BaGet/blob/master/src/BaGet/Entities/SqliteContext.cs) for how SQLite does it.